### PR TITLE
#8764: Additional set of changes for WH readiness, including new installation steps

### PR DESCRIPTION
**Pull request by tt-rkim**
### Ticket

#8764 

### Problem description

More complete installation steps for WH single card, and probably T3000.

### What's changed

Update docs.

### Checklist
- [ ] Post commit CI passes
- [ ] Model regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -583,9 +583,9 @@ cat generated/watcher/watcher.log  # See k_ids field for each core in the last d
 If a Tenstorrent chip seems to hang and/or is producing unexpected behaviour,
 you may try a software reset of the board.
 
-For Grayskull: `tt-smi -tr all`
+For single-card: `tt-smi -r 0`
 
-For Wormhole: `tt-smi -wr all wait`
+For T3000 (QuietBox, LoudBox etc.): `tt-smi -r 0,1,2,3`
```
Comment (ayerofieiev-tt) [inline CONTRIBUTING.md:588]: any chance tt-smi team can add support for `-r all`?

**Review by ayerofieiev-tt — APPROVED** [review]

**Review by dvartaniansTT — CHANGES_REQUESTED** [review]
I confirm build from whl from main with the latest firmware works now. `[fw_pack-80.10.0.0.fwbundle](https://github.com/tenstorrent/tt-firmware/blob/main/fw_pack-80.10.0.0.fwbundle)`

However, v50 tag will fail with the FW and main will fail with the previous firmware. We tested this on a N150 machine. 

Also please note my comment on the review regarding adding instructions for CMAKE install which is necessary to build from src.

File: INSTALLING.md [hunk 2]
```diff
@@ -90,39 +94,33 @@ cd tt-metal
...
 
 We use CMake for our build flows.
 
+Set up the environment variables and invoke our build scripts. Note that for
+source builds, you must set these environment variables every time.
+
 ```sh
+export ARCH_NAME=<ARCH_NAME>
+export TT_METAL_HOME=$(pwd)
+export PYTHONPATH=$(pwd)
```
[the remark this answers is not in the document]
Comment (tt-rkim) [reply INSTALLING.md:111]: Please check now

**Comment by tt-rkim** [comment]
@dvartaniansTT We will focus on cutting a release soon.
@uaydonat @skhorasganiTT any chance we can get a release candidate to line up? Recall we should be treating demo models drifting / failing from main as P0s.

**Comment by skhorasganiTT** [comment]
@tt-rkim If you need one now, it looks like the last set of demo runs (single-card and t3k) on main (d8957c8) passed. Could a release be generated from that? Alternatively, we can have one for our next round of model updates on Monday. 
Also unrelated observation, it looks like the package and release workflow schedule is now slightly out of sync with single-card/t3k demos. Should it be aligned?

**Comment by tt-rkim** [comment]
@skhorasganiTT No need to be aligned, unless you want to make sure every pipeline you look at for those is at the same time. It grabs the commit at which the tag is being generated - which makes sense. You have to test at the tag

**Comment by tt-rkim** [comment]
I have started a release generation. Unfortunately I haven't gotten the package and release changes approved yet to start a run of the demos on the same pipeline, but once the tag is made, I will kick off the demo pipelines on that tag. I can show you in person what that looks like @skhorasganiTT

**Review by dvartaniansTT — APPROVED** [review]
LGTM. thanks @tt-rkim

**Pull request merged**